consensus/src/dag/types.rs

```diff
@@ -298,3 +290,172 @@ pub struct DAGNetworkMessage {
     #[serde(with = "serde_bytes")]
     pub data: Vec<u8>,
 }
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub enum DAGMessage {
```

Comment:
so many boilerplate 🤦 but I don't see a simpler way to do it better, maybe we can do a procmacro but not today